pkg/apis/core/types_shoot.go

```diff
@@ -863,8 +863,13 @@ type KubeletConfig struct {
 	MaxPods *int32
 	// PodPIDsLimit is the maximum number of process IDs per pod allowed by the kubelet.
 	PodPIDsLimit *int64
+	// TODO(shafeeqes): Remove this field in gardener v1.89
```

Comment:
```suggestion
	// TODO(shafeeqes): Remove this field in gardener v1.90
```

https://github.com/gardener/gardener/pull/9129 got meanwhile merged.
Same comment applies for the v1beta1 types.

Reply:
I have asked @timuthy whether we can include this in v1.87.

Reply:
I guess you rather mean v1.88?

Reply:
Ah, yes. Sorry.